style.css

```diff
@@ -359,3 +371,110 @@ h2 {
   justify-content: center;
   margin-left: 220px;
 }
+
+.briefings {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-container {
+  background-color: #fff;
+  width: 100%;
+  margin: 0;
+}
+
+.contact-me {
+  background-color: #6070ff;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  border-top-left-radius: 100px;
+  background-image: url("contact form shapes-mobile@2x.svg");
+  background-repeat: no-repeat;
+  background-position: right;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
I can see you added a background image to the contact form, but it is not working. Kindly check the link and fix it and make sure it is working just as in the figma design.

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-28 18-56-04](https://user-images.githubusercontent.com/65233860/192866448-327047dc-7a4e-49a6-9c28-3e60fb0f4291.png)  |  ![Screenshot from 2022-09-28 18-55-01](https://user-images.githubusercontent.com/65233860/192866487-7437cbba-5219-4f70-b70a-128e9ac81781.png)